### PR TITLE
add guide to import swagger into postman

### PR DESCRIPTION
**Pull request by B-Naoufal**

### CODE REVIEW

File: postman_swagger.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Use postman with the swagger spec 
+
+**1. Locate the last version of swagger.yml file under `api/swagger/server/restapi/resource/swagger.yml`**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/ede19fc2-fd2f-4290-a432-87c7907278ac/ascreenshot.jpeg?tl_px=0,450&br_px=1493,1290&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=132,139)
+
+**2. Copy the whole content of `swagger.ym`l**
```
Comment (Thykof) [inline postman_swagger.md:7]:
```suggestion
**2. Copy the whole content of `swagger.ym`**
```

File: postman_swagger.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+**10. Now Postman will detect MassaStation API and it's type.**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/06d61be3-6162-4f4b-a17d-6731b36e174f/ascreenshot.jpeg?tl_px=1530,381&br_px=3023,1221&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=282,139)
+
+**11. Click Import and wait**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/9c6f9e99-8fab-4198-9de7-208ceb950e4d/ascreenshot.jpeg?tl_px=585,617&br_px=2078,1457&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=262,139)
+
+**12. When Import is Done, you will be able to see MassaStation APi imported, and you can browse all it's endpoints**
```
Comment (Thykof) [inline postman_swagger.md:47]:
```suggestion
**12. When Import is Done, you will be able to see MassaStation API imported, and you can browse all it's endpoints**
```

File: postman_swagger.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+**15. replace the `{{baseUrl}}` by `station.massa`**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/7afc05d9-5cf5-4985-86f2-683a314b879e/ascreenshot.jpeg?tl_px=399,48&br_px=1892,888&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=262,139)
+
+**16. click on send, and wait few seconds ...**
+
+![](https://ajeuwbhvhr.cloudimg.io/colony-recorder.s3.amazonaws.com/files/2023-07-11/51e5c1f9-b4fd-4a60-b90a-fd72c29a57d5/ascreenshot.jpeg?tl_px=1530,68&br_px=3023,908&force_format=png&width=560&wat_scale=50&wat=1&wat_opacity=0.7&wat_gravity=northwest&wat_url=https://colony-recorder.s3.us-west-1.amazonaws.com/images/watermarks/FB923C_standard.png&wat_pad=446,139)
+
+**17. You will receive in response all websites uploaded so far to the current DNS.**
```
Comment (Thykof) [inline postman_swagger.md:68]: this endpoint take a while to respond maybe it's not the best one to demonstrate Massa Station. @SlnPons WDTY?

**Review by Thykof — CHANGES_REQUESTED** [review]

File: postman_swagger.md (hunk 3 above)
Comment (B-Naoufal) [reply postman_swagger.md]: we are working on it to fix the delay , but it's the more representative endpoint to get some significant data ..

File: docs/postman_swagger.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+![](images/postman_swagger/14.png)
+
+15. **Replace the `{{baseUrl}}` by `station.massa` or define it as an environment variable in Postman:**
+   
+   For more flexibility, define `baseUrl` as your own environment variable in Postman. This allows you to easily switch between different environments or server instances without modifying the requests individually.
+   
+   Refer to the [Postman documentation](https://learning.postman.com/docs/sending-requests/variables/) for details on defining and managing environment variables in Postman.
+
+
```
Comment (Thykof) [inline docs/postman_swagger.md:65]:
```suggestion
You can edit the collection by clicking on the "..." menu, then go to the Variable tab, and add the variable.
```

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by Thykof — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Comment by SlnPons** [comment]
@B-Naoufal how do we move forward with this issue please? 
More than a week to write such a simple documentation is worrying me... Can I help you with anything?

**Pull request closed without merging**